src/App.jsx

```diff
@@ -1,14 +1,13 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.css';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum a={2} b={3} />
+    <Sum a={-5} b={5} />
+    <Sum a={10} b={0} />
+    <Sum a={0} b={5} />
+    <Sum a={0} b={0} />
```

Comment:
<img width="411" alt="Screenshot 2022-11-29 at 17 22 15" src="https://user-images.githubusercontent.com/94289038/204569665-6aa7caa9-ca76-4801-8595-8a9c80f8144b.png">
0 !== absence of property